snapshot_queries/stacktrace.py

```diff
@@ -58,16 +59,15 @@ def load(
 
     @staticmethod
     def _get_module_path(module_name) -> typing.Optional[str]:
-        try:
-            package = pkgutil.get_loader(module_name)
-        except ImportError:
-            return None
-        if not package:
+        module_spec = importlib.util.find_spec(module_name)
+        if module_spec is None or module_spec.origin is None:
             return None
 
-        source_path = package.path
+        source_path = module_spec.origin
+
         if source_path.endswith("__init__.py"):
             source_path = os.path.dirname(source_path)
+
```

Comment:
pytest modifies what is returned by `pkgutil.get_loader`, so this code had to be updated